src/components/dashboard/ShareLink.js

```diff
@@ -1,13 +1,32 @@
+// @flow
+import { isMobile } from 'mobile-device-detect'
 import React, { useCallback } from 'react'
+import { Clipboard } from 'react-native'
 
 import logger from '../../lib/logger/pino-logger'
 import { CustomButton as Button } from '../common'
 
 const log = logger.child({ from: 'ShareLink' })
 
-const ShareLink = ({ children, ...props }) => {
-  const link = props.link
-  const share = useCallback(() => log.warn(link))
+type Props = {
+  children: any,
+  link: string,
+  props: any
+}
+
+const ShareLink = ({ children, link, ...props }: Props) => {
+  const share = useCallback(() => {
+    if (isMobile) {
+      navigator.send({
```

Comment:
1. probably should be navigator.share
2. not all mobile browsers support .share, need to check if navigator.share exists. I think this was implemented somewhere else also.

Reply:
@sirpy thanks for the catch. Fix at: https://github.com/GoodDollar/GoodDAPP/pull/106/commits/6eaae4e67ff853f0c10b1ab99e81a63ec30fb863